### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created wish list page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  height: 3rem;
+  padding: 0 1.25rem;
+  font-size: 1rem;
+  border-radius: 1rem;
+  border: none;
+  background-color: var(--slate-700);
+  outline: none;
+  color: var(--white);
+}
+.search__bar:focus {
```
Comment (j24m) [inline index.css:22]: Use beautify extension to have consistent spaces between each class .

File: index.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+        <a class="prev" onclick="plusSlides(-1)">❮</a>
+        <a class="next" onclick="plusSlides(1)">❯</a>
+      </div>
+
+      <div class="mt-5" style="text-align: center">
+        <span
+          class="dot"
+          style="width: 10px; height: 10px"
+          onclick="currentSlide(1)"
```
Comment (j24m) [inline index.html:153]: Hey, it's not recommended to use inline js, maybe you can make a separate js file later. :)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+        ></span>
+        <span
+          style="width: 10px; height: 10px"
+          class="dot"
+          onclick="currentSlide(2)"
```
Comment (j24m) [inline index.html:158]: And here too, and everywhere else where you have used inline js or css both.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+
+    <!-- main -->
+    <main>
+      <!-- featured brands -->
+      <h3 class="h3 font-semibold text-center text-slate-900 mb-1">
+        Featured Brands
+      </h3>
+      <!-- <div class="w-96 bg-rose-500 h-1 m-auto"></div> -->
```
Comment (j24m) [inline index.html:180]: If there is no use for  this code here maybe you can remove it?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+      <!-- featured categories -->
+      <h3 class="h3 font-semibold text-center text-slate-900 mb-1">
+        Featured Categories
+      </h3>
+      <!-- <div class="w-96 bg-rose-500 h-1 m-auto"></div> -->
+      <div class="spacer-3rem"></div>
+      <div class="spacer-3rem"></div>
+
+      <div
+        class="grid grid-cols-4 w-full m-auto"
```
Comment (j24m) [inline index.html:220]: Please do not use inline style.

**Review by j24m — APPROVED** [review]
Hey, In your code you have used inline js and inline css at a lot of places, in case you have time left you can make the changes now or later, your choice, please provide with a screenshot or live link of the UI, so that user can parallely check both.Other than that class names are good.
I appreciate the efforts and work , keep it up :)

File: index.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
+var slideIndex = 1;
+slides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
```
Comment (VedanthB) [inline index.js:5]: can use arrow functions here

**Review by VedanthB — APPROVED** [review]
Good job ananya , everything looks good to go 🚀

File: index.css (hunk 1 above)
Comment (Ananyamadhu08) [reply index.css:22]: okay

File: index.html (hunk 2 above)
Comment (Ananyamadhu08) [reply index.html:153]: I have added inline styles so the classes from my component lib are overwritten , i wanted a few properties to overwrite the styles being applied from my component lib. As inline styles are more specific i have used them here

File: index.html (hunk 3 above)
Comment (Ananyamadhu08) [reply index.html:158]: I have added inline styles so the classes from my component lib are overwritten , i wanted a few properties to overwrite the styles being applied from my component lib. As inline styles are more specific i have used them here

File: index.html (hunk 4 above)
Comment (Ananyamadhu08) [reply index.html:180]: okay will make the changes

File: index.html (hunk 5 above)
Comment (Ananyamadhu08) [reply index.html:220]: I have added inline styles so the classes from my component lib are overwritten , i wanted a few properties to overwrite the styles being applied from my component lib. As inline styles are more specific i have used them here

File: index.js (hunk 6 above)
Comment (Ananyamadhu08) [reply index.js:5]: Okay will make the changes

**Pull request merged**